src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryIT.java

```diff
@@ -239,6 +262,28 @@ public void selectAliases() throws IOException {
         });
     }
 
+    @Test
+    public void useTableAliasInWhereClauseTest() throws IOException {
+        JSONObject response = executeQuery(String.format(Locale.ROOT,
+            "SELECT * FROM %s/account a WHERE a.city = 'Nogal' LIMIT 1000", TEST_INDEX_ACCOUNT));
+
+        JSONArray hits = getHits(response);
+        Assert.assertEquals(1, getTotalHits(response));
+        Assert.assertEquals("Nogal", getSource(hits.getJSONObject(0)).get("city"));
```

Comment:
Can we also add another test or assert that term query for city field has keyword alias appended.

Reply:
Sure, adding.